cmd/juju/commands/common.go

```diff
@@ -112,19 +112,19 @@ func environFromNameProductionFunc(
 	return env, cleanup, err
 }
 
-// resolveCharmURL resolves the given charm URL string
+// resolveEntityURL resolves the given charm or bundle URL string
```

Comment:
I agree, this is a good point. resolveCharmStoreEntityURL seems good to me, renamed.